src/commands/coin/coin.ts

```diff
@@ -10,7 +17,266 @@ import {
+const infoEmbed = new MessageEmbed()
+  .setColor(EMBED_COLOUR)
+  .setTitle('🪙   About Codey Coin   🪙')
+  .setThumbnail('https://emojipedia-us.s3.dualstack.us-west-1.amazonaws.com/thumbs/240/twitter/282/coin_1fa99.png') // Thumbnail, if needed?
+  .setDescription(`Codey coins are rewarded for being active in CSC's events and discord!`)
+  .addFields(
+    {
+      name: '🪙   How Can I Obtain Codey Coins?',
+      value: `Earn Codey coins by:
+      • Participating in CSC events
+      • Being active on Discord
+      ---Daily bonus - your first message of the day on CSC's Discord will grant some Codey coins
+      ---Activity bonus - your first message of every 5 minutes on CSC's Discord will grant some Codey coins`
+    },
+    {
+      name: '🪙   What Can I Do With Codey Coins?',
+      value: `Use Codey coins to:
+      • Play Casino games such as Blackjack
+      • Buy in-server CSC Swag (more info to come!)`
+    }
+  );
+
+const coinInfoExecuteCommand: SapphireMessageExecuteType = async (
+  _client,
+  _messageFromUser,
+  _args
+): Promise<SapphireMessageResponse> => {
+  return infoEmbed;
+};
+
+const coinInfoCommandDetails: CodeyCommandDetails = {
+  name: 'info',
+  aliases: ['information, i'],
+  description: 'Get info about CodeyCoin.',
+  detailedDescription: `**Examples:**
+\`${container.botPrefix}coin info\`
+\`${container.botPrefix}coin information\`
+\`${container.botPrefix}coin i\``,
+
+  isCommandResponseEphemeral: false,
+  messageWhenExecutingCommand: 'Getting information about coin:',
+  executeCommand: coinInfoExecuteCommand,
+  codeyCommandResponseType: CodeyCommandResponseType.EMBED,
+
+  options: [],
+  subcommandDetails: {}
+};
+
+// Update coin balance of a user
+const coinUpdateExecuteCommand: SapphireMessageExecuteType = async (client, messageFromUser, args) => {
+  if (!(<Readonly<Permissions>>messageFromUser.member?.permissions).has('ADMINISTRATOR')) return '';
+
+  // First mandatory argument is user
+  const user = <User>args['user'];
+  if (!user) {
+    throw new Error('please enter a valid user mention or ID for balance adjustment.');
+  }
+
+  // Second mandatory argument is amount
+  const amount = args['amount'];
+  if (!amount) {
+    throw new Error('please enter a valid amount to adjust.');
+  }
+
+  // Optional argument is reason
+  const reason = args['reason'];
+
+  // Adjust coin balance
+  await updateCoinBalanceByUserId(
+    user.id,
+    <number>amount,
+    UserCoinEvent.AdminCoinAdjust,
+    <string>(reason ? reason : ''),
+    client.user?.id
+  );
+  // Get new balance
+  const newBalance = await getCoinBalanceByUserId(user.id);
+
+  return `${user.username} now has ${newBalance} Codey coins 🪙.`;
+};
+
+const coinUpdateCommandDetails: CodeyCommandDetails = {
+  name: 'update',
+  aliases: ['u'],
+  description: 'Update the coin balance of a user.',
+  detailedDescription: `**Examples:**
+\`${container.botPrefix}coin update @Codey 100\``,
+
+  isCommandResponseEphemeral: true,
```

Comment:
For a similar reasoning as above, I think this should be false as well.

Reply:
Sounds good, I've also added this